api/tests/e2e/processes_test.go

```diff
@@ -0,0 +1,74 @@
+package e2e_test
+
+import (
+	"net/http"
+
+	"github.com/go-resty/resty/v2"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	rbacv1 "k8s.io/api/rbac/v1"
+)
+
+var _ = Describe("Processes", func() {
+	var (
+		orgGUID     string
+		spaceGUID   string
+		appGUID     string
+		processGUID string
+		result      resource
+		list        resourceList
+		resp        *resty.Response
+	)
+
+	BeforeEach(func() {
+		orgGUID = createOrg(generateGUID("org"))
+		DeferCleanup(func() {
```

Comment:
I personally find `DeferCleanup` a bit hard to follow as I would not expect cleanup code in `BeforeEach`. I would suggest moving org deletion into a `AfterEach`